FE/src/pages/Landing/Foreigner/RecommendPanel.tsx

```
@@ -0,0 +1,61 @@
+  RPDocument,
+  RPMessage,
+  RPPeople,
+} from "../../../assets/icon/RecommendPanelIcon";
+import { IcPencilLine } from "../../../assets/icon/StratisUi";
+import Button from "../../../components/common/Button";
+
+const RecommendPanel = () => {
+  return (
+    <section className="flex flex-row mt-7 p-10 border-[1.5px] border-violet-200 rounded-2xl">
+      {recommendations.map((rec) => (
+        <div className="flex flex-col w-85 gap-4">
```

Biome (2.3.13)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)